wazuhapp/Docker/build.sh

```diff
@@ -2,83 +2,119 @@
+        echo "Error downloading package.json from GitHub."
+        exit 1
+    fi
+
+    wazuh_version=$(python -c 'import json, os; f=open("/tmp/package.json"); pkg=json.load(f); f.close();\
+                    print(pkg["version"])')
+    kibana_version=$(python -c 'import json, os; f=open("/tmp/package.json"); pkg=json.load(f); f.close();\
+                     print(pkg["kibana"]["version"])')
+
+    kibana_package_json_url="${kibana_app_raw_repo_url}/v${kibana_version}/package.json"
 
-    if ! curl -O ${package_json_url} ; then
+    echo "Downloading package.json from elastic/kibana repository"
+    if ! curl $kibana_package_json_url -o "/tmp/package.json" ; then
         echo "Error downloading package.json from GitHub."
         exit 1
     fi
 
-    wazuh_version=$(python -c 'import json, os; f=open("/package.json"); pkg=json.load(f); f.close(); print(pkg["version"])')
-    kibana_version=$(python -c 'import json, os; f=open("/package.json"); pkg=json.load(f); f.close(); print(pkg["kibana"]["version"])')
+    kibana_node_version=$(python -c 'import json, os; f=open("/tmp/package.json"); pkg=json.load(f); f.close();\
+                          print(pkg["engines"]["node"])')
+
+    kibana_yarn_version=$(python -c 'import json, os; f=open("/tmp/package.json"); pkg=json.load(f); f.close();\
+                          print(pkg["engines"]["yarn"])')
 
     {
-        node_version=$(python -c 'import json, os; f=open("/package.json"); pkg=json.load(f); f.close(); print(pkg["node_build"])')
+        wazuh_app_node_version=$(python -c 'import json, os; f=open("/tmp/package.json"); pkg=json.load(f); f.close();\
+                                 print(pkg["node_build"])')
     }||{
-        node_version=$(python -c 'import json, os; f=open("/package.json"); pkg=json.load(f); f.close(); print(pkg["node"])')
+        wazuh_app_node_version=$(python -c 'import json, os; f=open("/tmp/package.json"); pkg=json.load(f); f.close();\
+                                 print(pkg["node"])')
     }||{
-        node_version="8.14.0"
+        wazuh_app_node_version="8.14.0"
     }
-
 }
 
-download_sources() {
 
-
-
-    if ! curl -L https://github.com/elastic/kibana/tarball/v${kibana_version} | tar zx ; then
-        echo "Error downloading Kibana source code from GitHub."
+download_kibana_sources() {
+    if ! git clone $kibana_app_repo_url --branch "v${kibana_version}" --depth=1 kibana_source; then
+        echo "Error downloading Kibana source code from elastic/kibana GitHub repository."
         exit 1
     fi
 
-    mv elastic-* kibana_source
     mkdir -p kibana_source/plugins
-
-    if ! git clone https://github.com/wazuh/wazuh-kibana-app.git --branch ${wazuh_branch} --depth=1 kibana_source/plugins/wazuh ; then
-        echo "Error downloading the source code from GitHub."
-        exit 1
-    fi
-
-    mv kibana_source ${kibana_dir}
 }
 
-install_dependencies (){
 
-    n ${node_version}
+install_dependencies () {
```

Comment:
This step is unnecessary because you install this version of the Node and then switch back to version 8.14.0 in the `build_package` method without performing any operations

Reply:
That's right, I thought you were changing the version of Node